src/router.js

```diff
@@ -51,7 +51,7 @@ class Router extends EventTargetShim {
     const a = e.composedPath().find((el) => el.tagName === 'A');
 
     if (!a || !a.href) return;
-    if (a.hasAttribute('download') || a.href.includes('mailto:')) return;
+    if (a.hasAttribute('download') || a.href.includes('mailto:') || a.href.includes('#')) return;
```

Comment:
this will block anchors to other domains with a #. I think skiplinks should still work without this, did you get any issues?

Reply:
Huh, weird. It was giving me full page refreshes earlier. I removed the check, and it still works